mage/installation.md

```diff
@@ -4,7 +4,7 @@ title: How to install MAGE?
 sidebar_label: Installation
 ---
 
-import Tabs from '@theme/Tabs'; 
+import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
 ## Prerequisites
```

Comment:
Could we recommend specific versions (or ranges) for **Python3**, **Make**, **CMake**, **Clang**, **UUID**, and
**Rust**?
It would be great if we could include a one-liner to install all of these at once. @jmatak

Reply:
Nice question, for now maybe we should just leave it as is, but the issue needs to be created on `memgraph/mage` to resolve this behavior since it is dependable on the system, and further investigation is needed.